### PR TITLE
service tests for [homebrew]

### PR DESCRIPTION
**Pull request by chris48s**

### CODE REVIEW

File: service-tests/homebrew.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  .networkOff()
+  .expectJSON({name: 'homebrew', value: 'inaccessible'});
+
+t.create('homebrew (unexpected response)')
+  .get('/v/cake.json')
+  .intercept(nock => nock('http://formulae.brew.sh')
+    .get('/formula/cake/version')
+    .reply(200, "{{{{{invalid json}}")
+  )
+  .expectJSON({name: 'homebrew', value: 'invalid'});
```
Comment (platan) [inline service-tests/homebrew.js]: How about adding a test which mocks and verifies the exact version?

```js
t.create('homebrew (valid exact version)')
  .get('/v/cake.json')
  .intercept(nock => nock('http://formulae.brew.sh')
    .get('/formula/cake/version')
    .reply(200, {stable: '0.23.0', devel: null, head: null})
  )
  .expectJSON({name: 'homebrew', value: 'v0.23.0'});
```

**Comment by platan** [comment]
Looks good to me. I've left one comment in code. Thanks for these tests!

**Comment by chris48s** [comment]
sure, why not.. :)

**Comment by paulmelnikow** [comment]
Fabulous. Thanks gents!

**Pull request merged**